crates/forge/src/collecting.rs

```diff
@@ -0,0 +1,590 @@
+use crate::{RunnerConfig, RunnerParams, BUILTINS};
+use anyhow::{anyhow, Context, Result};
+use assert_fs::fixture::{FileTouch, PathChild, PathCopy};
+use assert_fs::TempDir;
+use cairo_lang_sierra::program::Program;
+use camino::{Utf8Path, Utf8PathBuf};
+use rayon::prelude::{IntoParallelRefIterator, ParallelIterator};
+use test_collector::{collect_tests, LinkedLibrary, TestCase};
+use walkdir::WalkDir;
+
+#[derive(Debug, Clone)]
+pub struct CompiledTests {
+    pub sierra_program: Program,
+    pub test_cases: Vec<TestCase>,
+    pub tests_location: CrateLocation,
+}
```

Comment:
This is also a bad name. It suggest that it contains only compiled code. It also doesn't give an information of the scope, is this an arbitrary set of tests? Or tests from a module.  Please rename it to `TestCrate`

Reply:
`CompiledTestCrate`, we have `TestCrate` struct already

Reply:
It has been renamed.

Reply:
I am ok with: `TestCrate`, `CrateWithTests` or `CompiledTestCrate`, your call @cptartur